components/brave_shields/browser/cookie_pref_service.cc

```diff
@@ -21,6 +23,21 @@ namespace brave_shields {
 
 namespace {
 
+void SetCookieWhitelistForSocialMedia(HostContentSettingsMap* map,
```

Comment:
this method makes more sense as `SetCookieWhitelist` because it will potentially add other pref based overrides, but `ForSocialMedia` seems irrelevant to the method and google auth isn't social media anyway. This is just setting a whitelist based on prefs